src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -455,16 +426,8 @@ public partial class TcpListener
         public bool ExclusiveAddressUse { get { return default(bool); } set { } }
         public System.Net.EndPoint LocalEndpoint { get { return default(System.Net.EndPoint); } }
         public System.Net.Sockets.Socket Server { get { return default(System.Net.Sockets.Socket); } }
-        public System.Net.Sockets.Socket AcceptSocket() { return default(System.Net.Sockets.Socket); }
         public System.Threading.Tasks.Task<System.Net.Sockets.Socket> AcceptSocketAsync() { return default(System.Threading.Tasks.Task<System.Net.Sockets.Socket>); }
-        public System.Net.Sockets.TcpClient AcceptTcpClient() { return default(System.Net.Sockets.TcpClient); }
         public System.Threading.Tasks.Task<System.Net.Sockets.TcpClient> AcceptTcpClientAsync() { return default(System.Threading.Tasks.Task<System.Net.Sockets.TcpClient>); }
-        public void AllowNatTraversal(bool allowed) { }
-        public System.IAsyncResult BeginAcceptSocket(System.AsyncCallback callback, object state) { return default(System.IAsyncResult); }
-        public System.IAsyncResult BeginAcceptTcpClient(System.AsyncCallback callback, object state) { return default(System.IAsyncResult); }
-        public static System.Net.Sockets.TcpListener Create(int port) { return default(System.Net.Sockets.TcpListener); }
-        public System.Net.Sockets.Socket EndAcceptSocket(System.IAsyncResult asyncResult) { return default(System.Net.Sockets.Socket); }
-        public System.Net.Sockets.TcpClient EndAcceptTcpClient(System.IAsyncResult asyncResult) { return default(System.Net.Sockets.TcpClient); }
         public bool Pending() { return default(bool); }
         public void Start() { }
         public void Start(int backlog) { }
```

Comment:
@josguil @pgavlin @SidharthNabar We probably want to remove `TransmitFileOptions` and the `SendPacketsFlags` property on `SocketAsyncEventArgs`.